src/Table.luna

```diff
@@ -863,13 +1356,180 @@ class Table:
         newTable
 
     def chart: ChartBuilder self
+
+    # Plots y versus x. Default style for plot is marker.
+    #
+    # > import Dataframes.Column
+    # > import Dataframes.Types
+    # > import Dataframes.Table
+    # >
+    # > def main:
+    # >     l1 = [1,2,3,4,5]
+    # >     l2 = [11,12,13,14,15]
+    # >     col1 = Column.fromList "col1" Int64Type l1
+    # >     col2 = Column.fromList "col2" Int64Type l2
+    # >     table = Table.fromColumns [col1 , col2]
+    # >     plot1 = table.plot "col1" "col2"
+    # >     None
+    #
+    # `columnName1`: X column name.
```

Comment:
I don't like "X column" — it sounds as if being X was a property of column.
Consider: "Column to be used for X-axis" (or even "column values")